lucene/src/test/java/org/hibernate/hql/lucene/test/LuceneQueryParsingTestBase.java

```diff
@@ -111,24 +111,24 @@ public void shouldCreateQueryWithUnqualifiedPropertyReferences() {
 	public void shouldCreateNegatedQuery() {
 		assertLuceneQuery(
 				"from IndexedEntity e where NOT e.name = 'same'",
-				"-name:same *:*" );
+				"-name:same #*:*" );
```

Comment:
This is the query generated when I upgrade the library and I don't change anything else. I thought it was due to the upgraded library and didn't think too much of it but it seems nobody knows why it is happening.